components/kserve/resources/servicemesh/kserve-inferencegraph-envoyfilter.tmpl.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
move it to https://github.com/opendatahub-io/opendatahub-operator/tree/main/controllers/components/kserve/resources/servicemesh